sematic/api/server.py

```diff
@@ -146,12 +185,7 @@ def make_log_config():
     switch_env(args.env)
 
     if args.debug:
-        socketio.run(
-            sematic_api,
-            port=get_config().port,
-            host=get_config().server_address,
-            debug=args.debug,
-        )
+        run_socketio(args.debug)
```

Comment:
In order to be symmetric with the else branch, and in order to be immediately obvious:
```suggestion
        run_socketio(True)
```